src/amo/components/Categories.js

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+
+import Link from 'amo/components/Link';
+import translate from 'core/i18n/translate';
+
+import './Categories.scss';
+
+
+export function filterAndSortCategories(categories, addonType, clientApp) {
+  return categories.filter((category) => {
```

Comment:
I find the implicit returns harder to follow and also just prefer all functions look the same for consistency. I find implicit return JS to be harder to scan.